cloud/amazon/ec2.py

```diff
@@ -240,6 +240,13 @@
     required: false
     default: null
     aliases: []
+  network_interfaces:
+    version_added: "2.0"
+    description:
+      - A list of existing network interfaces to attach to the instance at launch. When specifying existing network interfaces, none of the assign_public_ip, private_ip, vpc_subnet_id, group, or group_id parameters may be used. (Those parameters are for creating a new network interface at launch.)
+    required: false
+    default: null
+    aliases: []
```

Comment:
Aliases don't need to be specified when there are none. You don't have to continue the bad habit of the original code.